### PR TITLE
Add Install-operators-verify ref

### PR DESCRIPTION
**Pull request by oharan2**

### CODE REVIEW

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-stolostron-acmqe-autotest-main-acm-ocp4.16-lp-interop-acm-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.16-lp-interop-mtc-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.16-lp-interop-mtc-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.16-lp-interop-mtc-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.16-lp-interop-mtc-interop-aws

File: ci-operator/step-registry/firewatch/ipi/aws/firewatch-ipi-aws-workflow.yaml [hunk 1]
```diff
@@ -2,7 +2,8 @@ workflow:
   as: firewatch-ipi-aws
   steps:
     pre:
-    - chain: ipi-aws-pre
+      - ref: install-operators-verify
```
Comment (calebevans) [inline ci-operator/step-registry/firewatch/ipi/aws/firewatch-ipi-aws-workflow.yaml:5]: My only issue with this is there are some scenarios that use the `firewatch-ipi-aws` workflow that do not use `install-operators` so the `OPERATORS` variable will not be present in the config. For example, [serverless](https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__ocp4.16-lp-interop.yaml#L217) does not use the ref (plus there are a couple of scenarios that use this that are not operators).

We may just need to add this to the scenarios where it applies? The grate thing about this being a ref is that we will be able to add a firewatch rule specifically for failures where the operator is not in the catalog for a pre-release OCP build. 😄

**Review by calebevans — CHANGES_REQUESTED** [review]

File: ci-operator/step-registry/firewatch/ipi/aws/firewatch-ipi-aws-workflow.yaml (hunk 1 above)
Comment (dfrazzette) [reply ci-operator/step-registry/firewatch/ipi/aws/firewatch-ipi-aws-workflow.yaml:5]: Can we define which scenarios will default to using the `firewatch-ipi-aws` workflow and apply this ref to them? Additionally, can we define which scenarios are exceptions (do not include `install-operators` ref; do not use operators, or use operators that may be in a different catalog)?

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.13-openshift-pipelines-ocp4.16-lp-interop-openshift-pipelines-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-skupperproject-skupper-openshift-smoke-test-image-main-service-interconnect-ocp4.16-lp-interop-rhsi-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-skupperproject-skupper-openshift-smoke-test-image-main-service-interconnect-ocp4.16-lp-interop-rhsi-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.13-openshift-pipelines-ocp4.16-lp-interop-openshift-pipelines-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-skupperproject-skupper-openshift-smoke-test-image-main-service-interconnect-ocp4.16-lp-interop-rhsi-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.13-openshift-pipelines-ocp4.16-lp-interop-openshift-pipelines-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-stolostron-acmqe-autotest-main-acm-ocp4.16-lp-interop-acm-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-quarkus-qe-quarkus-openshift-interop-main-quarkus-ocp4.16-lp-interop-quarkus-interop-aws

**Review by calebevans — APPROVED** [review]
/lgtm

File: ci-operator/step-registry/firewatch/ipi/aws/firewatch-ipi-aws-workflow.yaml (hunk 1 above)
Comment (oharan2) [reply ci-operator/step-registry/firewatch/ipi/aws/firewatch-ipi-aws-workflow.yaml:5]: Providing info for future interest:

Within our internal discussions, we verified that using the `OPERATORS` env var is 100% optional, therefore scenarios that uses the `firewatch-ipi-aws` workflow and apply this ref doesn't get effected

See POC:
Job [periodic-ci-quarkus-qe-quarkus-openshift-interop-main-quarkus-ocp4.16-lp-interop-quarkus-interop-aws](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/50255/rehearse-50255-periodic-ci-quarkus-qe-quarkus-openshift-interop-main-quarkus-ocp4.16-lp-interop-quarkus-interop-aws/1775150998822588416#1:build-log.txt%3A34)
- Uses the workflow
- Doesn't use `install-operators` ref for installing the Operator
- Passed this step with:
`Step quarkus-interop-aws-install-operators-verify succeeded after 8s.`
(Job aborted manually before executing cluster provision)

**Pull request merged**